docs/source/models/causes/neonatal/preterm_birth.rst

```diff
@@ -202,7 +209,64 @@ where :math:`k` is the subcause of interest (preterm birth with or without RDS),
 
-Each individual simulant :math:`i` has their own :math:`\text{CSMR}_i^k` that might be different from :math:`\text{CSMRisk}^k_{\text{BW}_i,\text{GA}_i}` (meaning the average birth-weight- and gestational-age-specific CSMRisk for simulants with the birth weight and gestational age matching simulant :math:`i`.  We recommend implementing this as a pipeline eventually because it will be modified by interventions (or access to interventions) relevant to this subcause.  (Until we implement those, we will have :math:`\text{CSMRisk}_{i}^k = \text{CSMRisk}^k_{\text{BW}_i,\text{GA}_i}`, though.)
+:math:`\text{CSMRisk}` and :math:`p_{\text{preterm}}` are calculated differently for the ENN and LNN age groups.
+For clarity of notation, in what follows we will again make explicit the age group
+subscripts that have been implicit on every quantity to this point.
+(Sex and location remain implicit.)
+We define the ENN CSMRisk as:
+
+.. math::
+
+  \text{CSMRisk}_\text{ENN} = \frac{\text{enn_death_count}}{\text{live_birth_count}}
+
+where the :math:`\text{enn_death_count}` and :math:`\text{live_birth_count}` are
+quantities pulled from GBD, as detailed in the table below.
+
+The LNN CSMRisk is:
+
+.. math::
+
+  \text{CSMRisk}_\text{LNN} = \frac{\text{lnn_death_count}}{\text{live_birth_count} - \text{enn_all_cause_death_count}}
+
+:math:`p_{\text{preterm}}`, as mentioned above, represents the prevalence/exposure
+of preterm (gestational age < 37 weeks) at the *beginning* of the age group.
+For ENN, the beginning of the age group is birth, so the prevalence of preterm
+at birth is a sum of the birth prevalence for all LBWSG categories with gestational
+age less than 37 weeks:
+
+.. math::
+
+  p_{\text{preterm},\text{ENN}} = \sum_{\{\text{cat}: \text{GA}<37\}} \text{lbwsg_birth_prevalence}_\text{cat}
+
+where :math:`\text{lbwsg_birth_prevalence}` can be pulled from GBD with minor transformations,
+as detailed in the table below.
+
+For LNN, the situation is more complicated, because we need to account
+for differential mortality in the ENN period.
+Therefore, the easiest way to calculate :math:`p_{\text{preterm},\text{LNN}}` is to get the end-of-ENN preterm
+prevalence from the same LBWSG PAF calculation pipeline used for :math:`Z`
+above.
+As detailed at :ref:`details_of_the_lbwsg_paf_calculation` on the neonatal all-cause
+mortality page, there are two iterative steps using microsimulation, with the late neonatal calculations
+using the result of the early neonatal calculations.
+Similarly to the LNN PAF, *after* the early neonatal calculations are complete, the prevalence of
+preterm at the end of the ENN age group should be calculated.
+This value should be used as :math:`p_{\text{preterm},\text{LNN}}` for the purposes
+of the CSMRisk equation.
+Concretely, in the PAF calculation pipeline, in which LBWSG categories have not been assigned using the birth prevalence (currently exactly equally between categories),
```

Comment:
I find this sentence confusing... Is this referring to the fact that in the PAF calculation sim, we initialize each category with the same number of simulants, rather than initializing each category with a number of simulants proportional to its birth prevalence? Is that documented somewhere? There must be a clearer way to get this across, maybe some way that explicitly mentions the "grid points" referred to parenthetically after the equation...

Reply:
Yes it is, I will revise to clarify!